docs/user-guide/ze-usage.md

```diff
@@ -0,0 +1,310 @@
+
+## Usage Tips
+
+Use the following tips to familiarize yourself with the extension and make the best use of it:
+
+* **Data set, USS and jobs persistence settings:** You can enable the persistence of any data sets, USS files and jobs by adding them to the **Favorites** tab. Right-click on a data set, USS file or jobs and click **Add Favorite**.
+
+* **Identify syntax errors wtih a syntax highlighter:** Zowe Explorer supports a syntax hightlighter for data sets. To enhance the experience of using the extension, download an extension that highlights syntax, such as [COBOL Language Support](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.cobol-language-support) or [HLASM Language Support](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.hlasm-language-support).
+
+* **Edit a profile**: You can now edit existing profiles by clicking the **pencil** button next to the **magnifying glass** button in the explorer tree. The feature lets you modify the information inside your profile.
+
+* **Delete a profile**: You can permanently delete profiles by right-clicking the profile and selecting the **Delete Profile** option. The feature deletes a profile from your `.zowe` folder. Alternatively, you can delete a profile from the VSCode Command Palette.
+
+  **Follow these steps:**
+
+   1. Press **F1** on your keyboard.
+   2. Select the **Zowe: Delete a Profile Permanently** option.
+
+* **Hide a profile**: You can hide a profile from profile tree by right-clicking the profile and selecting the **Hide Profile** option. If necessary, add the profile back by clicking the **+** button from the explorer tree.
+
+## Sample Use Cases
+
+Review the following use cases to understand how to use Zowe Explorer.
+
+* [Work with Data Sets](#work-with-data-sets)
+* [Work with USS Files](#work-with-uss-files)
+* [Work with JOBS](#work-with-jobs)
+* [MVS/TSO Commands](#mvs/tso-commands)
+
+### Work with Data Sets
+
+You can use the following functionalities when interacting with data set:
+
+* **View data sets and use multiple filters**: You can view multiple data sets simultaneously and apply filters to show specified data sets.
+* **Rename data sets**: You can rename specified data sets.
+* **Copy data sets**: You can copy a specified data sets and memebers.
+* **Download, edit, and upload existing PDS members**: You can instantly pull data sets and data set members from the mainframe, edit them and upload back.
+* **Create and delete data sets and data set members**: Enables you to easily create and delete both data sets and their members.
+* **View and access multiple profiles simultaneously**: Enables to work with data sets from multiple profiles.
+* **Submit a JCL**: You can submit a jcl from a chosen data set.
```

Comment:
"Submit a job"  

(while Job Control Language just happens to be the language the job is written in).

Reply:
This one is based on the UX title. I agree with you 100% on this one.